_layouts/layout.html

```diff
@@ -82,6 +82,12 @@
     </script>
     {% endif %}
 
+      
+
+  <!-- Used for LATEX equations -->
+<script type="text/javascript" src="http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
+<script type="text/javascript" src="/js/jq_mathjax_parse.js"></script>
```

Comment:
Ahh, makes sense. I have made the changes. It is now using your mechanism of mathjax!